src/Admin/AmpPlugins.php

```diff
@@ -274,7 +274,7 @@ public function filter_plugins_api( $response, /** @noinspection PhpUnusedParame
 		$response->info    = [
 			'page'    => $page,
 			'pages'   => $total_page,
-			'results' => count( $plugins ),
+			'results' => count( $this->get_plugins() ),
```

Comment:
Oh, I see what I did wrong when I changed this. I didn't account for `$plugins` being overridden. Ooops.